resources/util.go

```diff
@@ -133,3 +135,35 @@ func LowerStructFieldNameToFieldName(res Res) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+// GetUID returns the UID of an user
+// Caller should first check user is not empty
+func GetUID(username string) (int, error) {
+	u2, err2 := user.LookupId(username)
```

Comment:
Is there a reason this is named `u2` and `err2`? If not, please use `err` and `uid` for example.
I realize it was weirdly named before, let's take the opportunity to fix it :)

Reply:
Done :) see https://github.com/purpleidea/mgmt/pull/256#discussion_r152585679